python/BUSINESS_SUPPORT_OR_ABOVE_ENABLED/business-support-enabled.py

```diff
@@ -0,0 +1,415 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.

```

Comment:
Please change the name of the file to BUSINESS_SUPPORT_OR_ABOVE_ENABLED.py

Reply:
file name changed